tests/code-splitting/subtests/dynamic-import-preloading/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Dynamic import preloading
+importance: 1
+shortDesc: 'Can lazy-loaded split points be preloaded?'
+---
+
+# Introduction
+
+When creating split points [using dynamic import](/code-splitting/dynamic-import/), it is possible to construct a "waterfall" scenario where JavaScript bundles depend on additional bundles discovered once executed. The ability to preload nested dependency bundles is important in these cases, since doing so can allow the dependency bundles to be downloaded and evaluated in parallel.
+
+# The Test
+
+This test checks to see if it's possible to request code-splitted dependency bundles before the consuming bundle has been loaded, either via [preload](https://web.dev/preload-critical-assets/) or async script loading.
+
+**index.js**
+
+```js
+setTimeout(async () => {
+  const { shout } =
+    Math.random() < 0.5
+      ? await import('./utilsA.js')
+      : await import('./utilsB.js');
+  shout('this is index');
+}, 6000);
```

Comment:
Is the `setTimeout` a necessary part of the test?

Reply:
I don’t believe it is.